test/expected/misc.out

```diff
@@ -231,3 +231,21 @@ ERROR:  Column `id` is already marked as an AID.
 CONTEXT:  PL/pgSQL function diffix.assert_column_is_not_aid(text,text) line 20 at RAISE
 SQL statement "CALL diffix.assert_column_is_not_aid(table_name, column_name)"
 PL/pgSQL function diffix.mark_filterable(text,text) line 3 at CALL
+-- JOIN between personal tables produces multiple AIDs
+EXPLAIN VERBOSE SELECT COUNT(*) FROM test_customers c JOIN test_purchases pur ON c.id = cid;
+                                         QUERY PLAN                                         
+--------------------------------------------------------------------------------------------
+ Aggregate  (cost=72.44..72.45 rows=1 width=8)
+   Output: count(*)
```

Comment:
Interesting; there is no BucketScan anywhere?

Reply:
Sorry, the test is wrong, because I mistakenly put it after a test that sets access to `direct` and didn't check the output in detail. Good catch! But the output of `EXPLAIN VERBOSE` differs between PostgreSQL 13 and 14. Any ideas on how to fix that?

Reply:
The difference seems to be caused by us setting the `queryId` field. PG 14 shows it, while PG 13 doesn't.
Is it ok with you if I remove that feature? It doesn't seem useful anyway.

Reply:
I remember we dropped some tests because of this discrepancy...
I was experimenting with a procedure that filters the id row, but dropping it altogether also works.

Reply:
Why do you want to keep the `queryId` field in the logs?

Reply:
> but dropping it altogether also works.

I'm not against deleting it, it has been useless so far, so feel free to.
It stuck around from the beginning where we thought we might need to attach data to queries and what not.

Reply:
Ok, then I will remove it, seems cleaner that way.